packages/components/src/components.d.ts

```diff
@@ -437,6 +447,11 @@ export namespace Components {
          */
         "readonly"?: boolean;
     }
+    /**
+     * es-progression demo.
+     */
+    interface EsProgressionDemo {
```

Comment:
On of the issues with the demos, is you then need to run `yarn build` before commiting, to not have them in this `components.d.ts`.